mne/time_frequency/tfr.py

```diff
@@ -158,7 +158,7 @@ def _make_dpss(sfreq, freqs, n_cycles=7., time_bandwidth=4.0, zero_mean=False):
 
             # Get dpss tapers
             tapers, conc = dpss_windows(t.shape[0], time_bandwidth / 2.,
-                                        n_taps)
+                                        n_taps, sym=False)
```

Comment:
Is this a bugfix? Or did our default for `dpss_windows` implicitly change? (If it's the latter, we should probably default to `sym=False` for backward compat)

Reply:
I consider this a bugfix. According to SciPy docs, symmetric windows should be used when filtering, and non-symmetric (periodic) windows should be used for spectral analysis.  (They don't point to a source for that though; maybe it's obvious to signal processing engineers?)  Our old code only made symmetric windows; now it's symmetric by default (for backward compat in case anyone used this func directly) and periodic just here (where we're doing spectral analysis). 

(AFAICT the difference is basically analogous to `endpoint=True/False` in `np.linspace` (at least when plotting tapers made with `sym=True/False` that's how it appears)).

So if you agree with that, I guess this would be another bugfix entry?

Reply:
Agreed this is also a bugfix. I'd combine the `latest.inc` entry with the `max_iter` one. The differences should be very small anyway